components/org.wso2.carbon.identity.workflow.engine/src/main/java/org/wso2/carbon/identity/workflow/engine/dto/ApprovalTaskRelationDTO.java

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright (c) 2025, WSO2 LLC. (http://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.carbon.identity.workflow.engine.dto;
+
+/**
+ * DTO class to represent an approval task data in WF_WORKFLOW_APPROVAL_RELATION.
+ */
+public class ApprovalTaskRelationDTO {
+
+    private String taskId = null;
+    private String eventId = null;
+    private String workflowId = null;
+    private String approverType = null;
+    private String approverName = null;
+    private String taskStatus = null;
+
+
+    public String getTaskId() {
```

Comment:
```suggestion
    public String getTaskId() {
    
```

Let's format this file.

Reply:
Addressed